Elsewhen/Extensions/TimeZone.swift

```diff
@@ -38,9 +42,8 @@ extension TimeZone {
     
     func matches(searchTerm: String) -> Bool {
         let st = searchTerm.trimmingCharacters(in: .whitespacesAndNewlines).lowercased()
-        if st == "" {
-            return true
-        }
+        guard st != "" else { return true }
+        let alternativeSearchTerm = st.replacingOccurrences(of: "_", with: " ")
```

Comment:
Is it worth making this an array of alternative search terms, replacing `_`, `-`, and `/` as a starter?

Reply:
This was a minor refactor of existing logic (it already did exactly this but putting it in its own variable means not calculating it twice later on) - an array could be a good approach but I'll do a separate MR for that?